tests/benchmark.py

```diff
@@ -312,23 +312,23 @@ def _get_random_layer(cfg: ngl.SceneCfg, rng, t0, t1, enable_computes, layer=4):
                 forward_transforms=True,
             )
             rtt.add_color_textures(rtt_tex)
-            rtt_render = ngl.RenderTexture(
+            rtt_draw = ngl.DrawTexture(
                 rtt_tex,
                 geometry=_get_random_geometry(rng),
                 blending="src_over",
             )
-            rtt_render = _get_random_transform(rng, t0, t1, rtt_render)
+            rtt_draw = _get_random_transform(rng, t0, t1, rtt_draw)
             t_start, t_end = _get_random_time_range(rng, t0, t1)
-            rtt_group = ngl.Group(children=(rtt, rtt_render))
+            rtt_group = ngl.Group(children=(rtt, rtt_draw))
             t_filter = ngl.TimeRangeFilter(rtt_group, t_start, t_end)
 
-            # Draw both the children and the rendered texture
+            # Draw both the children and the drawed texture
```

Comment:
drawn

Reply:
Kept `renderered`, it makes reference to the rtt.